### ISSUE TITLE
Snow theme player slider wrong color on Chrome

### ISSUE THREAD

**Issue by Nicolas-Gth** [loof2736/scyfin#38]
It seems that you forgot to replace the colors in the playback-video.css file.
![image](https://github.com/user-attachments/assets/f1baabbd-5d30-479e-b085-fb28b138c7c9)

My fix for chrome (but the shape is still not the same as Firefox and I would be better to use the --accent variable rather than hardcoding the color hex).
```CSS
.mdl-slider::-webkit-slider-thumb {
              background: #fff;
}
```
![image](https://github.com/user-attachments/assets/1b1a1525-5353-4014-aab8-f487effde8f5)

I saw many other classes with the hard coded blue color that I think should be replaced with the --accent variable
```CSS
.countIndicator, .fullSyncIndicator, .mediaSourceIndicator, .playedIndicator {
    background: #fff;
}
```
for example

**Comment by loof2736** [comment]
This seems to be a bug with some new version of Jellyfin not handling the CSS correctly if they have both Webkit and Firefox specific selectors in the same CSS block. I'll see if I can find which version this originated in and open a bug report on the main Jellyfin github repo. 

CSS like this works in Firefox, but not Chrome:

.mdl-slider::-moz-range-thumb,
.mdl-slider::-webkit-slider-thumb {
    background: blue !important;
}

For Chrome, it has to be formatted like this for some reason:

.mdl-slider::-moz-range-thumb {
    background: blue !important;
}
.mdl-slider::-webkit-slider-thumb {
    background: blue !important;
}

**Comment by loof2736** [comment]
Actually this doesn't seem to be an issue with Jellyfin, rather CSS in general. I could have sworn this was working before, but I must have moved some stuff around in the CSS and never tested it in Chrome.

Basically Chrome doesn't recognize certain Firefox specific CSS selectors (and vice versa, I just haven't run into any in Firefox). When this happens, Chrome just throws out the entire CSS block they are present in.

I just need to separate a couple selectors, should be a simple fix

**Comment by loof2736** [comment]
Fixed in v1.4.8
